### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by Gandalf47** [naturalcrit/homebrewery#3234] [Q&A] [answered]
Is there any way to download images and use those? I'm having a difficult time using images from just online links.

**Comment by 5e-Cleric** [comment, marked as answer]
As per right now, local images cannot be used in the Homebrewery, as other people would not be able to access the file on the share page, among other problems.

This is likely going to stay like that for some time.

We reccomend hosting your own images on imgur, or other image hosting services.
